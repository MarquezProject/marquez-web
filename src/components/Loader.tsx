import * as React from 'react'
import CircularProgress from '@material-ui/core/CircularProgress'

interface IProps {}

export default (props: IProps) => {
  return <CircularProgress color='secondary' />
}
