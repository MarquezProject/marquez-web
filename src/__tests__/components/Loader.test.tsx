import * as React from 'react'
import { mount } from 'enzyme'
import Loader from '../../components/Loader'

describe('Loader Component', () => {
  const wrapper = mount(<Loader />)

  it('renders a snapshot that matches previous', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
