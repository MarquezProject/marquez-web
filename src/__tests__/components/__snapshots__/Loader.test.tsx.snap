// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader Component renders a snapshot that matches previous 1`] = `
<WithStyles(CustomLoader)>
  <CustomLoader
    classes={
      Object {
        "container": "CustomLoader-container-1",
      }
    }
  >
    <div
      className="CustomLoader-container-1"
    >
      <Styled(WithStyles(ForwardRef(CircularProgress)))
        color="secondary"
      >
        <WithStyles(ForwardRef(CircularProgress))
          className="WithStyles(ForwardRef(CircularProgress))-root-2"
          color="secondary"
        >
          <ForwardRef(CircularProgress)
            className="WithStyles(ForwardRef(CircularProgress))-root-2"
            classes={
              Object {
                "circle": "MuiCircularProgress-circle",
                "circleDisableShrink": "MuiCircularProgress-circleDisableShrink",
                "circleIndeterminate": "MuiCircularProgress-circleIndeterminate",
                "circleStatic": "MuiCircularProgress-circleStatic",
                "colorPrimary": "MuiCircularProgress-colorPrimary",
                "colorSecondary": "MuiCircularProgress-colorSecondary",
                "indeterminate": "MuiCircularProgress-indeterminate",
                "root": "MuiCircularProgress-root",
                "static": "MuiCircularProgress-static",
                "svg": "MuiCircularProgress-svg",
              }
            }
            color="secondary"
          >
            <div
              className="MuiCircularProgress-root WithStyles(ForwardRef(CircularProgress))-root-2 MuiCircularProgress-colorSecondary MuiCircularProgress-indeterminate"
              role="progressbar"
              style={
                Object {
                  "height": 40,
                  "width": 40,
                }
              }
            >
              <svg
                className="MuiCircularProgress-svg"
                viewBox="22 22 44 44"
              >
                <circle
                  className="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate"
                  cx={44}
                  cy={44}
                  fill="none"
                  r={20.2}
                  strokeWidth={3.6}
                  style={Object {}}
                />
              </svg>
            </div>
          </ForwardRef(CircularProgress)>
        </WithStyles(ForwardRef(CircularProgress))>
      </Styled(WithStyles(ForwardRef(CircularProgress)))>
      <Styled(WithStyles(ForwardRef(CircularProgress)))
        color="secondary"
        size={80}
      >
        <WithStyles(ForwardRef(CircularProgress))
          className="WithStyles(ForwardRef(CircularProgress))-root-2"
          color="secondary"
          size={80}
        >
          <ForwardRef(CircularProgress)
            className="WithStyles(ForwardRef(CircularProgress))-root-2"
            classes={
              Object {
                "circle": "MuiCircularProgress-circle",
                "circleDisableShrink": "MuiCircularProgress-circleDisableShrink",
                "circleIndeterminate": "MuiCircularProgress-circleIndeterminate",
                "circleStatic": "MuiCircularProgress-circleStatic",
                "colorPrimary": "MuiCircularProgress-colorPrimary",
                "colorSecondary": "MuiCircularProgress-colorSecondary",
                "indeterminate": "MuiCircularProgress-indeterminate",
                "root": "MuiCircularProgress-root",
                "static": "MuiCircularProgress-static",
                "svg": "MuiCircularProgress-svg",
              }
            }
            color="secondary"
            size={80}
          >
            <div
              className="MuiCircularProgress-root WithStyles(ForwardRef(CircularProgress))-root-2 MuiCircularProgress-colorSecondary MuiCircularProgress-indeterminate"
              role="progressbar"
              style={
                Object {
                  "height": 80,
                  "width": 80,
                }
              }
            >
              <svg
                className="MuiCircularProgress-svg"
                viewBox="22 22 44 44"
              >
                <circle
                  className="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate"
                  cx={44}
                  cy={44}
                  fill="none"
                  r={20.2}
                  strokeWidth={3.6}
                  style={Object {}}
                />
              </svg>
            </div>
          </ForwardRef(CircularProgress)>
        </WithStyles(ForwardRef(CircularProgress))>
      </Styled(WithStyles(ForwardRef(CircularProgress)))>
    </div>
  </CustomLoader>
</WithStyles(CustomLoader)>
`;
